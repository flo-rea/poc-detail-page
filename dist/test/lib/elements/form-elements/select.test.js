"use strict";
const React = require('react');
const renderer = require('react/lib/ReactTestRenderer');
const select_1 = require('../../../../src/lib/view/elements/form-elements/select');
const select_option_1 = require('../../../../src/lib/view/elements/form-elements/select-option');
jest.mock('react-dom');
describe('Select', () => {
    const component = renderer.create(React.createElement(select_1.Select, {id: "item2", name: "item2"}, 
        React.createElement(select_option_1.Option, null, "My first option"), 
        React.createElement(select_option_1.Option, null, "My second option"), 
        React.createElement(select_option_1.Option, null, "My third option")));
    it('has correct default options', () => {
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
//# sourceMappingURL=select.test.js.map