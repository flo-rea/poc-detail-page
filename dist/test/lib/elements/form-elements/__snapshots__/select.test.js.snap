exports[`Select has correct default options 1`] = `
<select
  id="item2"
  name="item2">
  <option>
    My first option
  </option>
  <option>
    My second option
  </option>
  <option>
    My third option
  </option>
</select>
`;
